package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "open-city-profile-ui",
-  "version": "1.1.11",
+  "version": "1.1.12",
```

Comment:
Should version changes also be reflected in the `public/env-config.js` file's `REACT_APP_VERSION` property? At least for me `yarn start` always wants to consolidate those values.

Reply:
Yeah, env-config.js is re-created every time server starts, so I don't see why it is in the repo. The process how the file is created, was done by Kuva, so I don't have first hand info.

There is a ticket (https://helsinkisolutionoffice.atlassian.net/browse/HP-1184) which repairs a bug in the process. The env-config.js could be added to .gitignore in that one. 

I'll ask Kuva developers why it is not ignored yet.

Reply:
Ok. So still within this PR, should the version number be changed in env-config.js file as well?

Reply:
I asked Kuva. env-config is in .gitignore, but as it is also in the repo so not ignored. It can be removed.

But anyway I added the file here as it technically should be here until fixed.